src/SFA.DAS.EmployerAccounts.Web/Extensions/UrlHelperExtensions.cs

```diff
@@ -94,6 +103,17 @@ private static string AccountAction(UrlHelper helper, string baseUrl, string pat
             return Action(baseUrl, accountPath);
         }
 
+        private static string CommitmentAction(UrlHelper helper, string baseUrl, string path)
+        {          
+            //var hashedAccountId = helper.RequestContext.RouteData.Values[ControllerConstants.AccountHashedIdRouteKeyName];
+            //var model = helper.RequestContext.RouteData.Values["model"]  as AccountDashboardViewModel;
+            //var hashedCohortReference = model?.CallToActionViewModel?.HashedCohortReference ?? string.Empty;
+            //var hashedDraftApprenticeshipId = model?.CallToActionViewModel?.HashedDraftApprenticeshipId ?? string.Empty;
+            //var commitmentPath = path == ControllerConstants.ApproveOrRejectApprentice ? $"{hashedAccountId}/unapproved/{hashedCohortReference}" : $"{hashedAccountId}/unapproved/{hashedCohortReference}/apprentices/{hashedDraftApprenticeshipId}";
+           
+            return Action(baseUrl, path);
```

Comment:
no need to pass on to this method. Can be deleted

Reply:
done